lib/puppet/ssl/host.rb

```diff
@@ -484,6 +478,30 @@ def load_certificate_bundle(bundle_string)
     end
   end
 
+  # Fetches the crl bundle from the CA server; because this is used for the first
+  # fetch of the crl, we need to pass in a certificate store that has no crls
+  # attached to it, but still utilizes the localcacert.
+  # @raise [Puppet::Error] if response from the server is not a valid certificate
+  #                        bundle
+  # @return nil
+  def download_and_save_crl_bundle
+    certificate_store = OpenSSL::X509::Store.new()
+    certificate_store.purpose = OpenSSL::X509::PURPOSE_ANY
+    certificate_store.add_file(Puppet.settings[:localcacert])
```

Comment:
Should we parameterize `build_ssl_store()` to take `use_crl?` and `use_crl_chain?` as options and then call that here?